### PR TITLE
[COMDLG32][REGEDIT] Correction of display problems (CORE-16890)

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

_ Current display of "key export" CommDlg is not rendering correctly due to a too short array in ComDlg

JIRA issue: [CORE-16890](https://jira.reactos.org/browse/CORE-16890)

image

## Proposed changes

_ First problem solved by extensing Array size from 16 to 24. Shared with WineHQ : https://bugs.winehq.org/show_bug.cgi?id=48962

image

### CODE REVIEW

**Comment by JoachimHenze** [comment]
The top change is good. But the bottom change is not. I attached a picture of XPSP3 to your ticket.
Please limit that PR to the top change maybe (and mention only one of both JIRA issues then ofc).

**Comment by KyleKatarn31** [comment]
> 
> 
> Please limit to the top left changes. The bottom left part should be fixed differently.

Done. OK to approve ?

File: dll/win32/comdlg32/filedlg.c [hunk 1]
```diff
@@ -1915,7 +1915,11 @@ static LRESULT FILEDLG95_InitControls(HWND hwnd)
   /* change Open to Save */
   if (fodInfos->DlgInfos.dwDlgProp & FODPROP_SAVEDLG)
   {
+#ifdef __REACTOS__
+	  WCHAR buf[24];
```
Comment (sdever) [inline dll/win32/comdlg32/filedlg.c:1919]: Mixed tabs/spaces

Comment (KyleKatarn31) [reply dll/win32/comdlg32/filedlg.c]: Fixed

File: dll/win32/comdlg32/filedlg.c [hunk 2]
```diff
@@ -1915,7 +1915,11 @@ static LRESULT FILEDLG95_InitControls(HWND hwnd)
   /* change Open to Save */
   if (fodInfos->DlgInfos.dwDlgProp & FODPROP_SAVEDLG)
   {
+#ifdef __REACTOS__
```
Comment (SergeGautherie) [inline dll/win32/comdlg32/filedlg.c:1918]: Fix upstream Wine first?

Comment (KyleKatarn31) [reply dll/win32/comdlg32/filedlg.c:1918]: What do you mean ?

Comment (SergeGautherie) [reply dll/win32/comdlg32/filedlg.c:1918]: Usual steps are:
1. Fix Wine
2. Sync ReactOS
3. Resolve Jira

Comment (KyleKatarn31) [reply dll/win32/comdlg32/filedlg.c:1918]: Yes, it has been reported to Wine : https://bugs.winehq.org/show_bug.cgi?id=48962

Comment (SergeGautherie) [reply dll/win32/comdlg32/filedlg.c:1918]: Are you unaware of the WineSync process?
Don't you understand the difference between a report filed in a bug tracker and a fix committed in a code repository? Scary...

Comment (KyleKatarn31) [reply dll/win32/comdlg32/filedlg.c:1918]: Yes i'm unaware of your processes. Happy to know that you'll help me.

Comment (SergeGautherie) [reply dll/win32/comdlg32/filedlg.c:1918]: I won't do more (clues) than I have already done, especially here. (See private chat.)

**Comment by KyleKatarn31** [comment]
> 
> 
> I think the PR is ok now and ready to be merged. Most likely we should contact the Wine devs also.
> Since DarkShadow44 is reading within CORE-16890 already, I hope it will be enough to make him aware again after we have committed this change.

Thanks ! Who could merge please ?

**Pull request merged**